resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -5,11 +5,12 @@ pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
 
 restartfile=<%= EZBake::Config[:restart_file] %>
 start_timeout=<%= EZBake::Config[:start_timeout] %>
-stop_timeout=60
 dir=$(dirname "$restartfile")
 realname="<%= EZBake::Config[:real_name] %>"
 PIDFILE="/var/run/puppetlabs/${realname}/${realname}.pid"
 
+[ -e "${INSTALL_DIR}/ezbake-functions.sh" ] && . "${INSTALL_DIR}/ezbake-functions.sh"
```

Comment:
no, `set +e` means don't fail on errors. `set -e` means fail on errors.

this line will not cause a failure if ezbake-functions.sh doesn't exist.